localstack/services/apigateway/helpers.py

```diff
@@ -625,8 +625,8 @@ def get_cors_response(headers):
     return response
 
 
-def get_rest_api_paths(rest_api_id, region_name=None):
-    apigateway = connect_to(region_name=region_name).apigateway
+def get_rest_api_paths(account_id: str, region_name: str, rest_api_id):
```

Comment:
It can be `None` from the perspective of the internal client library (`connect_to`). However, we want to be as explicity as possible with account IDs and regions to keep expectable behaviour with cross-account usage :)

Reply:
Type hint added in 9ef697da